### PR TITLE
Feature/service activation and configuration

### PR DESCRIPTION
**Pull request by Jason337**

### CODE REVIEW

File: config/application_routes.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 const express = require('express')
 const routes = express.Router()
 
-//  === Controllers ===
+//  ===   Default   ===
```
Comment (GarvieMichael) [inline config/application_routes.js]:
```suggestion
//  ===   Routes   ===
```

File: config/application_routes.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ routes.use("/usage_specification", require("./routes/usage_specification_routes"
 routes.use("/usage_hub", require("./routes/usage_hub_routes"))
 routes.use("/roles", require("./routes/roles_routes"))
 routes.use("/permissions", require("./routes/permissions_routes"))
+routes.use("/service_act_and_conf", require("./routes/service_activation_and_configuration_routes"))
```
Comment (GarvieMichael) [inline config/application_routes.js]:
```suggestion
routes.use("/sac", require("./routes/service_activation_and_configuration_routes"))
```

File: config/routes/service_activation_and_configuration_routes.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+service_activation_and_configuration_routes.delete('/service/:id', (request, response) => {
+    response.send( service_activation_and_configuration_controller.remove_service(request) )
+})
+service_activation_and_configuration_routes.get('/monitor', (request, response) => {
+    response.send( service_activation_and_configuration_controller.index_monitor(request) )
+})
+service_activation_and_configuration_routes.get('/monitor/:id', (request, response) => {
+    response.send( service_activation_and_configuration_controller.show_monitor(request) )
+})
+service_activation_and_configuration_routes.post('/listener/notification/monitor_create', (request, response) => {
```
Comment (GarvieMichael) [inline config/routes/service_activation_and_configuration_routes.js]:
```suggestion
service_activation_and_configuration_routes.post('/notification/monitor/create', (request, response) => {
```

File: app/controllers/service_activation_and_configuration_controller.js [hunk 4]
```diff
@@ -0,0 +1,1085 @@
...
+            ],
+            supportingService: [
+                {
+                    id: faker.random.uuid(),
+                    href: faker.internet.url()
+                }
+            ]
+        }
+    },
+    remove_service: function(request)
```
Comment (GarvieMichael) [inline app/controllers/service_activation_and_configuration_controller.js]:
```suggestion
    destroy_service: function(request)
```

Comment (GarvieMichael) [reply app/controllers/service_activation_and_configuration_controller.js]: Apply this to all the remove_xxxx please

**Review by GarvieMichael — CHANGES_REQUESTED** [review]

File: config/application_routes.js (hunk 1 above)
Comment (Jason337) [reply config/application_routes.js]: fixed

File: config/application_routes.js (hunk 2 above)
Comment (Jason337) [reply config/application_routes.js]: updated

File: config/routes/service_activation_and_configuration_routes.js (hunk 3 above)
Comment (Jason337) [reply config/routes/service_activation_and_configuration_routes.js]: changed

**Review by GarvieMichael — APPROVED** [review]

**Pull request merged**